packages/tokens/src/tokens/core/fonts.tokens.json

```diff
@@ -0,0 +1,124 @@
+{
+    "font": {
+        "family": {
+            "primary" : {
+                "$type": "string",
+                "$value": "ABC Favorit"
+            },
+            "secondary" : {
+                "$type": "string",
+                "$value": "Inter"
+            },
+            "tertiary" : {
+                "$type": "string",
+                "$value": "ABC Favorit Mono"
+            }
+        },
+        "size": {
+            "120": {
+                "$type": "number",
+                "$value": "12"
+            },
+            "140": {
+                "$type": "number",
+                "$value": "14"
+            },
+            "160": {
+                "$type": "number",
+                "$value": "16"
+            },
+            "180": {
+                "$type": "number",
+                "$value": "18"
+            },
+            "200": {
+                "$type": "number",
+                "$value": "20"
+            },
+            "240": {
+                "$type": "number",
+                "$value": "24"
+            },
+            "280": {
+                "$type": "number",
+                "$value": "28"
+            },
+            "320": {
+                "$type": "number",
+                "$value": "32"
+            },
+            "360": {
+                "$type": "number",
+                "$value": "36"
+            },
+            "480": {
+                "$type": "number",
+                "$value": "48"
+            }
+        },
+        "weight": {
+            "690": {
+                "$type": "number",
+                "$value": "690"
+            },
+            "680": {
+                "$type": "number",
+                "$value": "680"
+            },
+            "590": {
+                "$type": "number",
+                "$value": "590"
+            },
+            "580": {
+                "$type": "number",
+                "$value": "580"
+            },
+            "505": {
+                "$type": "number",
+                "$value": "505"
+            },
+            "410": {
+                "$type": "number",
+                "$value": "410"
+            },
+            "400": {
+                "$type": "number",
+                "$value": "400"
+            }
+        }
+    },
+    "line-height": {
+        "112-5": {
+            "$type": "number",
+            "$value": "1.125"
+        },
+        "114": {
+            "$type": "number",
+            "$value": "1.14"
+        },
+        "120": {
+            "$type": "number",
+            "$value": "1.2"
+        },
+        "125": {
+            "$type": "number",
+            "$value": "1.25"
+        },
+        "133": {
+            "$type": "number",
+            "$value": "1.33"
+        },
+        "142-8": {
```

Comment:
I think we are overcomplicating things here, 142-8 means 1.4285 if we are going for a 1:1 naming convention shouldn't this be 142-85? Thinking about our styled system I wonder if this makes sense: `<Text lineHeight={142-8} ...` At a minimum I would go with the dash replacing the dot meaning `142-8` would become `1-428` or `1-4285`.  1.33 would be `1-33`